cli/src/pcluster/cli/commands/cluster.py

```diff
@@ -37,6 +37,26 @@
 LOGGER = logging.getLogger(__name__)
```

Comment:
Do you want to call the new one?

I was going to do the removal and cut-over to new commands as a separate pass.

Reply:
As you prefer, ListClustersCommand is not used anywhere cause is from V3 and not V2 so feel free to do the transition right away.

What about displaying the available choices in the helper? is this something reasonable to do?

Reply:
We should be showing choices now. This was blocked by the "> 4" metavar code since there are so many options, but that's since been removed and the help now shows the options.